sources/packages/backend/libs/test-utils/src/factories/education-program.ts

```diff
@@ -18,7 +18,6 @@ export function createFakeEducationProgram(relations?: {
   program.credentialType = "credentialType";
   program.cipCode = "cipCode";
   program.nocCode = "nocCode";
-  program.sabcCode = "sabcCode";
```

Comment:
Why was sabcCode removed?

Reply:
`sabcCode` must be unique (when provided). This code was generating a DB exception if called twice.